Resources/Prototypes/_Goobstation/Catalog/Fills/Belt/belts.yml

```
@@ -266,3 +268,23 @@
     - id: SupermatterGrenade
     - id: SyndieMiniBomb
       amount: 2
+
+- type: entity
+  id: ClothingBeltMilitaryWebbingBSOFiled
+  parent: ClothingBeltMilitaryWebbingBSO
+  suffix: Filled, BSO
+  components:
+  - type: StorageFill
+    contents:
+    - id: CombatInjector
+      amount: 1
+    - id: TirimolChemistryVial
+      amount: 1
+    - id: OmnizineChemistryVial
+      amount: 1
+    - id: SyriniverChemistryVial
+      amount: 1
+    - id: ProcenylLazideChemistryVial
+      amount: 1
+    - id: EbifinlLazideChemistryVial
+      amount: 1
```

The default integer value for amount is 1.
```suggestion
    - id: CombatInjector
    - id: TirimolChemistryVial
    - id: OmnizineChemistryVial
    - id: SyriniverChemistryVial
    - id: ProcenylLazideChemistryVial
    - id: EbifinlLazideChemistryVial
```